src/unity/python/turicreate/visualization/show.py

```diff
@@ -96,46 +96,302 @@ def show(x, y, xlabel="X", ylabel="Y", title=None):
     Plot(plt_ref).show()
 
 def scatter(x, y, xlabel="X", ylabel="Y", title=None):
+    """
+    Plots the data in `x` on the X axis and the data in `y` on the Y axis
+    in a 2d scatterplot, and returns the resulting Plot object.
+    
+    The function supports SArrays of dtypes: int, float.
+
+    Parameters
+    ----------
+    x : SArray
+      The data to plot on the X axis of the scatter plot. 
+      Must be numeric (int/float)
```

Comment:
Please put a period at the end (complete sentences encouraged but not required).